### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by an0d3va**
Formatted to work with github pages.

### CODE REVIEW

**Comment by georgezgeorgez** [comment]
going to add this to the readme, but just fyi
https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll

File: dart-sdk-docs/examples/included-example.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## Imports 
+
+    import 'package:hex/hex.dart';
+    import 'package:znn_sdk_dart/znn_sdk_dart.dart';
+
+This example imports the whole SDK in one go, but it uses only functions from KeyStore and KeyPair.
+
+## Main part
+
+    This is the main routine of the example:
```
Comment (georgezgeorgez) [inline dart-sdk-docs/examples/included-example.md:25]: Did you mean to indent this and include it in the code block?

**Comment by an0d3va** [comment]
> 
> 
> going to add this to the readme, but just fyi https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/testing-your-github-pages-site-locally-with-jekyll

Oh thank you for that, that's much better

**Review by georgezgeorgez — APPROVED** [review]
LGTM
I haven't actually installed dart (lol) or run any of the examples yet
but i figure it is better to get this in front of the community asap for their use and feedback

**Pull request merged**